cmd/juju/commands/upgradejuju.go

```diff
@@ -337,7 +337,7 @@ func (c *upgradeJujuCommand) Run(ctx *cmd.Context) (err error) {
 		if c.BuildAgent {
 			builtMsg = " (built from source)"
 		}
-		fmt.Fprintf(ctx.Stdout, "no prepackaged tools available, using local agent binary %v%s\n", context.chosen, builtMsg)
+		fmt.Fprintf(ctx.Stdout, "no prepackaged agent binaries available, using local agent binary %v%s\n", context.chosen, builtMsg)
```

Comment:
wow, we actually said "no tools  ... using agent binary", go us :)

Reply:
\o/ we try ...